qpc/scan/utils.py

```diff
@@ -21,6 +21,11 @@
 
 
 def _get_source_ids(parser, source_names):
+    """Grab the source ids from the source if it exists.
```

Comment:
would name this function get_source_ids because it is used outside of the util module.